### DISCUSSION TITLE
Hide other players drops on ground plugin (it would be awesome for ironmans)

### DISCUSSION THREAD

**Discussion by Birkki** [runelite/runelite#16459] [Ideas]
I play ironman and when on task you see alot items on ground and if you are afk you dont know what drop is yours so you try to pick it up but you cant. This plugin could benefit alot ironmans and it would add good amount of quality to the gameplay.

**Comment by Alexsuperfly** [comment]
the ground items plugin has the "only show loot" option that does this

**Reply by Birkki** [reply]
Oh didnt know, thanks!
